### PR TITLE
Create scoreboard

### PR DESCRIPTION
**Pull request by oscarahalvorsen**
### PR Type 🍏
What kind of change does this PR introduce?
```
[ ] Bugfix
[X] Feature
[ ] API
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```
[Kanban Link](insert here)

### Context 🤷‍♀️
We needed a scoreboard 

### What's new? 👶
Created scoreboard.
It contains exception handling for when there is: 
1. event not yet loaded
2. actually no event
3. an event but with no players

Also added pagination to handle many players

### Screenshots 🖼️
image
image
image
image
image
image

image

### CODE REVIEW

**Comment by oscarahalvorsen** [comment]
NB! Only works if you are in add-get-active-event branch or have merge it into main already

**Comment by oscarahalvorsen** [comment]
Please review the backend code first

**Review by maribsta — APPROVED** [review]
Bra jobbet! Jeg skal bare dobbelsjekke at prettier er aktivert før den merges

File: src/app/(dark)/leaderboard/[difficulty]/page.tsx
[the remark this answers is not in the document]
Comment (oscarahalvorsen) [reply src/app/(dark)/leaderboard/[difficulty]/page.tsx:1]: La den tilbake

**Pull request merged**